docs/operator-guide/troubleshooting/resource-observability.md

```diff
@@ -0,0 +1,37 @@
+# Resource Observability Issue
+
+## Problem
```

Comment:
do we need to highlight that we have two approaches for this? OIDC and static token